vignettes/ci.Rmd

```diff
@@ -97,15 +97,12 @@ The following template can be used as a base when using renv with
 
 ```
 variables:
-  RENV_CONFIG_REPOS_OVERRIDE: "http://cran.r-project.org"
-  RENV_PATHS_CACHE: ${CI_PROJECT_DIR}/cache
-  RENV_PATHS_LIBRARY: ${CI_PROJECT_DIR}/renv/library
+  RENV_PATHS_CACHE: ${CI_PROJECT_DIR}/renv/cache
```

Comment:
Putting in `/renv/cache` rather than `/cache` removed the need to add the `cache/` directory to the project's `.Rbuildignore` file.